### ISSUE TITLE
Mini player cover image not centered

### ISSUE THREAD

**Issue by breferrari** [peterprd/Grupi-reports-and-request#60]
I've tried using the mini player and it works just fine, but the cover art is not centered on the notification.

![photo_2020-05-26_16-00-31](https://user-images.githubusercontent.com/1744013/82909849-1c65ba00-9f6a-11ea-838e-41dcd1a7c390.jpg)

**Comment by 3b1eab48ab86** [comment]
same

**Comment by chrisfenger** [comment]
Yes I also have this issue with the mini player artwork from grupi on iPhone 7 13.5.

**Comment by opiper** [comment]
I have this too. iPhone 11 on iOS 13.5

**Comment by ghost** [comment]
![7238EA5B-DB9E-4B77-89CB-44C3AAC96BAC](https://user-images.githubusercontent.com/50657918/84511847-0c1a4280-acbf-11ea-8c55-9eb57f36af62.png)

The same on my iPhone XR iOS 13.5.
